### PR TITLE
MAT-8442: Add missing information

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-8442](https://jira.cms.gov/browse/MAT-8442)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included and Code coverage is verified.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/cqllibraryservice/services/LibrarySetService.java [hunk 1]
```diff
@@ -218,7 +224,11 @@ public LibrarySet updateOwnership(
           String.format("Transferred from %s to %s", originalOwner, userId));
       if (retainShareAccess) {
         actionLogService.logShareAccessControlAction(
-            librarySet.getLibrarySetId(), ActionType.SHARED, conductedBy, originalOwner);
+            librarySet.getLibrarySetId(),
+            ActionType.SHARED,
+            conductedBy,
+            originalOwner,
+            String.format("Transferred from %s to %s", originalOwner, userId));
```
Comment (etan-sb) [inline src/main/java/gov/cms/madie/cqllibraryservice/services/LibrarySetService.java]: If we are logging a SHARED action, shouldn't we be logging a message like:

`String.format("Shared with - %s", userId));`

Comment (mcmcphillips) [reply src/main/java/gov/cms/madie/cqllibraryservice/services/LibrarySetService.java]: You're right

**Review by adongare — APPROVED** [review]

**Review by etan-sb — APPROVED** [review]
